api/libproject/devstart.rb

```diff
@@ -1060,6 +1060,38 @@ def backfill_gsuite_user_data(cmd_name, *args)
   :fn => ->(*args) {backfill_gsuite_user_data("backfill-gsuite-user-data", *args)}
 })
 
+def fetch_firecloud_user_profile(cmd_name, *args)
```

Comment:
You might argue that the "Fetch user profile" command-line tool is useless, which it is – but it did force me to ensure the creds impersonation actually works from the command-line. And it provides a simple pattern for anyone who needs to write a tool which actually does something useful w/ the FireCloud APIs.

Reply:
If the goal is to provide that pattern for people who want to write a tool which does something useful with the FireCloud APIs, can we remove the `register_command` block? Just to avoid unused commands showing up in the blank arg `./project.rb`?

Reply:
Is it possible to run the command at all if I leave it out? Part of the imagined utility here would be that we can let someone see that it actually works.

Maybe adding a prefix to the command like "example_" or "tool_example" could help make it obvious in the command list?

Either way... project.rb is a huge mess and I'm hoping https://precisionmedicineinitiative.atlassian.net/browse/RW-605 will give us a better model for maintaining many commands in an organized state.